docs/docusaurus/versioned_docs/version-1.2.0/lte/enodebd.md

```diff
@@ -163,3 +163,29 @@ failure until you add the corresponding IMSI to the subscriber database.
 
 Continue to the next section to register your subscribers and configure your
 APNs to start serving traffic.
+
+## Unmanaged eNB Installation via AGW
+
+### Background
+
+The configuration of unmanaged (eNBs lacking TR-069 functionality) is done through the dnsd service within the AGW. In order to enable dynamic IP allocation of these eNBs, this service must be disabled.
+ 
+### Disable dnsd Service
+
+1. SSH into your AGW 
+2. Type ```cat -n dnsd.conf```
+3. Comment out the line for dhcp-range 
```

Comment:
This isn't durable because dnsd.conf will get overridden the next time dnsd restarts. We control the generation of this config from Orchestrator via the network's DNS config, you can flip DHCPServerEnabled to false on the API. But this will apply to all AGW's in your network, so maybe we need to make this a gateway-specific config now.